legend-depot-store-mongo/src/main/java/org/finos/legend/depot/store/mongo/BaseVersionMongo.java

```diff
@@ -0,0 +1,181 @@
+package org.finos.legend.depot.store.mongo;
```

Comment:
u dont need to do this, project version data  implements hasidentifier so you mongo dao only has to extend base mongo